### ISSUE TITLE
Add more context for the different links in the website.

### ISSUE THREAD

**Issue by SaptakS** [freedomofpress/securedrop.org#873]
Add more context for the different links in the website. For example, instead of just writing Release notes & Documentation & Read more, make it clear what the context of the link is. Also, same texts should have same link, and same link shouldn't have different text.

Solution:
we can add aria-label to describe the link  without needing to change the actual text of the link.

`<a href="https://docs.securedrop.org" aria-label="Documentations about SecureDrop">Documentation</a>`

So the minimum things to do while fixing through the links are:
- [ ] Add `aria-label` with proper context about the link if the link text is generic, or it is same all over. For example "View Directory.." link, or Read More links, etc.
- [ ] If same links have different text, add `aria-label` for the links which are same describing what the link is pointing to.

WCAG [2.4.4](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-refs.html), [2.4.9](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-link.html)
